docker/secondary-analysis-python/Dockerfile

```diff
@@ -5,3 +5,9 @@ LABEL software="Python"
 LABEL description="Python with some useful libraries"
 
 RUN pip install requests
+
+RUN mkdir /tools
+
+COPY tools /tools
```

Comment:
If you think this is mature enough to release an alpha version, then lets put this package up on pypi; you'll need to update setup.py to include tags see: [unfortunately-also-named-cromwell-tools](https://github.com/ambrosejcarr/cromwell_tools/blob/master/setup.py), but otherwise there isn't much overhead. The result will be you no longer need to copy in a local folder, and can pip install it.

Reply:
That's a great point! I wasn't sure whether to keep this as a separate package, or include it as part of #19 so I just kept it local for now